requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not needed in the repository, consider providing .gitignore

Reply:
Done